less/home-page.less

```
@@ -52,10 +102,26 @@
     border-top: 2px dashed @navigation-border;
     display: flex;
     justify-content: space-between;
+    @media (max-width: 500px) {
```

Would have liked to have seen an interpolated string.
```suggestion
    @media @phone {
```